### PR TITLE
More ITIM error-handling improvements

### PR DESCRIPTION
**Pull request by aswaterman**

### CODE REVIEW

File: src/main/scala/rocket/ICache.scala [hunk 1]
```diff
@@ -313,7 +313,7 @@ class ICacheModule(outer: ICache) extends LazyModuleImp(outer)
         }
 
         respValid := s2_slaveValid || (respValid && !tl.d.ready)
-        val respError = RegEnable(s2_scratchpad_hit && s2_data_decoded.uncorrectable, s2_slaveValid)
+        val respError = RegEnable(s2_scratchpad_hit && s2_data_decoded.uncorrectable && !(edge_in.get.hasData(s1_a) && s1_a.mask.andR), s2_slaveValid)
```
Comment (terpstra) [inline src/main/scala/rocket/ICache.scala]: Is s1_a the right stage in the pipeline for this?

Comment (aswaterman) [reply src/main/scala/rocket/ICache.scala]: yes it's ok

**Comment by terpstra** [comment]
There's something wrong with this PR. It times out in the unit test regressions, despite being restarted.

**Comment by aswaterman** [comment]
it's rebuilding the tools, which I assume is why it's timing out...

**Comment by terpstra** [comment]
Ok. Looks like the tools were bumped wrong. Blew out the cache and restarting this PR for the purpose of correcting the cache.

**Comment by aswaterman** [comment]
thanks @terpstra

**Pull request merged**